### PR TITLE
Add release notes for 3.3.30-3 and 3.4.5-3

### PR DESCRIPTION
**Pull request by timtheisen**
I have clicked on the links to JIRA and koji and the IGTF change log. All looks in order to me.

### CODE REVIEW

File: docs/release/3.3/release-3-3-30-3.md [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+#### Enterprise Linux 7
+
+-   [igtf-ca-certs-1.88-1.osg33.el7](https://koji.chtc.wisc.edu/koji/search?match=glob&type=build&terms=igtf-ca-certs-1.88-1.osg33.el7)
+-   [osg-ca-certs-1.68-1.osg33.el7](https://koji.chtc.wisc.edu/koji/search?match=glob&type=build&terms=osg-ca-certs-1.68-1.osg33.el7)
+
+### RPMs
+
+If you wish to manually update your system, you can run yum update against the following packages:
+
+    igtf-ca-certs osg-ca-certs osg-gums-config vo-client vo-client-edgmkgridmap vo-client-lcmaps-voms
```
Comment (brianhlin) [inline docs/release/3.3/release-3-3-30-3.md]: There isn't a VO update is there? Same goes for the other doc

Comment (timtheisen) [reply docs/release/3.3/release-3-3-30-3.md]: Good catch. I pushed an update with the requested changes.

Comment (timtheisen) [reply docs/release/3.3/release-3-3-30-3.md]: Still missing a change. Ugh.

File: docs/release/3.3/release-3-3-30-3.md [hunk 2]
```diff
@@ -0,0 +1,82 @@
...
+
+``` console
+root@host # yum update
+```
+
+!!! note
+    Please be aware that running `yum update` may also update other RPMs. You can exclude packages from being updated using the `--exclude=[package-name or glob]` option for the `yum` command.
+
+!!! note
+    Watch the yum update carefully for any messages about a `.rpmnew` file being created. That means that a configuration file had been editted, and a new default version was to be installed. In that case, RPM does not overwrite the editted configuration file but instead installs the new version with a `.rpmnew` extension. You will need to merge any edits that have made into the `.rpmnew` file and then move the merged version into place (that is, without the `.rpmnew` extension). Watch especially for `/etc/lcmaps.db`, which every site is expected to edit.
```
Comment (brianhlin) [inline docs/release/3.3/release-3-3-30-3.md]: I just discussed this with @matyasselmeci and we don't need to stress `/etc/lcmaps.db` anymore. Same goes for 3.3

**Comment by timtheisen** [comment]
It should be fine now.

**Comment by timtheisen** [comment]
How about now?

**Pull request merged**